### PR TITLE
Update installation instructions in the README

### PR DESCRIPTION
**Pull request by sashaperigo**
I expanded the installation instructions in the README in order to address a few pain points I had while setting up badssl.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,21 +18,29 @@ Stock Ubuntu VM, DNS A records for `badssl.com.` and `*.badssl.com.` pointing to
...
-The root CA is at `certs/sets/test/gen/crt/ca-root.crt`. If you'd like to preserve it even when you run `make clean`, run:
+5. You can now navigate to `badssl.test` in your browser, and you should see a certificate error.
 
-    cd certs/sets/test
-    mkdir -p pregen/crt pregen/key
-    cp gen/crt/ca-root.crt pregen/crt/ca-root.crt
-    cp gen/key/ca-root.key pregen/key/ca-root.key
+6. The badssl root certificate is at `certs/sets/test/gen/crt/ca-root.crt`. In order to get the rest of the badssl subdomains working, you will need to add this to your machine's list of trusted certificates.
+
+##### Mac OS X
```
Comment (lgarron) [inline README.md]: Nit: macOS

Comment (lgarron) [reply README.md]: Also, could you make this a sub-bullet of 6? Otherwise, I think some Markdown parsers will renumber #7 to #6.

**Comment by sashaperigo** [comment]
I signed it!

File: README.md [hunk 2]
```diff
@@ -18,21 +18,29 @@ Stock Ubuntu VM, DNS A records for `badssl.com.` and `*.badssl.com.` pointing to
...
 ### Testing and development
 
-Your need to [install Docker](https://www.docker.com/get-docker) to run badssl.com locally.
+1. Follow the instructions to [install Docker.](https://www.docker.com/get-docker)
 
-    git clone https://github.com/chromium/badssl.com && cd badssl.com
-    make list-hosts # list of domains to copy into /etc/hosts
+2. Clone into the badssl repo by running `git clone https://github.com/chromium/badssl.com && cd badssl.com`.
+ 
+3. In order to access the various badssl subdomains locally you will need to add them to your [system hosts file](http://bencane.com/2013/10/29/managing-dns-locally-with-etchosts/). Run `make list-hosts` and copy and paste the output into `/etc/hosts`.
```
Comment (lgarron) [inline README.md]: Nit: link to HTTPS version of that page. ;-)

Comment (sashaperigo) [reply README.md]: Oops! Done.

File: README.md (hunk 1 above)
Comment (sashaperigo) [reply README.md]: Good idea! I moved it to a sub-bullet.

**Comment by lgarron** [comment]
Thanks!

**Pull request merged**